conf/custom-dme-hubble.conf

```diff
@@ -0,0 +1,169 @@
+#################################
```

Comment:
This is just a sample config which will be removed eventually and actually sample config etc would be added in respective files.

This file is added to provide an example configuration that can be used to test osqueryd_monitor and osqueryd_log_parser functions.